system/lvol.py

```diff
@@ -212,7 +221,11 @@ def main():
                 changed = True
             else:
                 lvcreate_cmd = module.get_bin_path("lvcreate", required=True)
-                rc, _, err = module.run_command("%s %s -n %s -%s %s%s %s" % (lvcreate_cmd, yesopt, lv, size_opt, size, size_unit, vg))
+                if opts is None:
```

Comment:
This doesn't really scale to multiple parameters. Other parameters have an earlier test to check if they should be used, otherwise they're just the empty string.

Take a look at `yesopt` that was added today for how I would prefer it.